system-variables.md

```diff
@@ -1106,6 +1106,9 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 
 > **Warning:**
 >
+>
+> Currently, this feature is not fully compatible with the [altering multiple columns or indexes in a single `ALTER TABLE` statement](/sql-statements/sql-statement-alter-table.md) feature. When adding a unique index with the index acceleration, you need to avoid altering other columns or indexes in the same statement.
```

Comment:
```suggestion
> Currently, this feature is not fully compatible with [altering multiple columns or indexes in a single `ALTER TABLE` statement](/sql-statements/sql-statement-alter-table.md). When adding a unique index with the index acceleration, you need to avoid altering other columns or indexes in the same statement.
```